types/trimblemaps__trimblemaps-js/index.d.ts

```diff
@@ -0,0 +1,2817 @@
+// Type definitions for @trimblemaps/trimblemaps-js 3.0
+// Project: https://developer.trimblemaps.com/maps-sdk/api/
+// Definitions by: Paito Anderson <https://github.com/PaitoAnderson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// THIS FILE WAS PARTIALLY COPIED FROM mapbox-gl
+
+/// <reference types="geojson" />
+
```

Comment:
```suggestion

export as namespace TrimbleMaps;

```
I think you need this one, global script support is explicitly mentioned on api docs:
https://developer.trimblemaps.com/maps-sdk/guide/getting-started/
and as I said, it's visible in UMD-type of the package itself:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library